src/__tests__/http-test.ts

```diff
@@ -2144,7 +2142,7 @@ function runTests(server: Server) {
         graphqlHTTP({
           schema: TestSchema,
           customFormatErrorFn: () => null,
-          extensions({ result }) {
+          extensions({ result }: { result: any }) {
```

Comment:
Why do you need this cast?

Reply:
`result` is unknown so it cannot be spread

Reply:
@danielrearden Can you please update result to be an `ExecutionResult` in a separate PR?